src/rgw/rgw_notify.cc

```diff
@@ -689,6 +689,7 @@ void populate_event_from_request(const reservation_t& res,
   event.object_size = size;
   event.object_etag = etag;
   event.object_versionId = version;
+  event.awsRegion = s->zonegroup_name;
```

Comment:
i think you probably want the zonegroup's `api_name` here. that's what we use for s3 apis that match a region name, for example CreateBucket's LocationConstraint: https://github.com/ceph/ceph/blob/master/src/rgw/rgw_op.cc#L2997-L3014

Reply:
is it because the `zonegroup_name` would be empty or because the `api_name` will have a different value?
also note that currently, we are not passing "store" to this function. we would need to do that if we want to use:
```
store->get_zone()->get_zonegroup().api_name
```

Reply:
api_name may have a different value intentionally, or the zonegroup may have been renamed

Reply:
@cbodley could you please explain what is the issue with using `zonegroup_name` here?

Reply:
only that it's not consistent with other APIs that expose a 'region name' to clients. that's the purpose of api_name

does it matter here? up to you. but if it's useful for the client to get a region name with notifications, it should be the one that they can use in requests

Reply:
ok. got it. thanks!
@arjune123 could you please change to the API value?